### PR TITLE
Creating and Editing Events

### PR DESCRIPTION
**Pull request by jeppes**
- [x] user permissions
- [x] uploading pictures

### CODE REVIEW

**Comment by jeppes** [comment]
@jberglinds please get this finished and merged by the way if you get the time. I think everything works, the design is very ugly. This feature is very important for the upcoming events though. I haven't really tested permissions either.

File: app/components/EventEdit/index.js [hunk 1]
```diff
@@ -1,174 +1,208 @@
...
+import { sortedUniq } from 'lodash'
+import { uploadImage } from 'api'
 import messages from './messages'
 import styles from './styles.css'
 import Textarea from 'react-textarea-autosize'
+import Button from 'components/Button'
+import EventEditPicture from 'components/EventEditPicture'
+import EventEditSurvey from 'components/EventEditSurvey'
 
+const MAX_PICTURE_COUNT = 3
```
Comment (aronstrandberg) [inline app/components/EventEdit/index.js]: Let's talk about this

File: app/components/EventEditPicture/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './styles.css'
+import { X } from 'react-feather'
+
+const EventEditPicture = ({ url, onRemove }) => (
+  <div className={styles.container}>
+    <img src={url}
+      className={styles.image}/>
+    <div className={styles.crossContainer}
```
Comment (aronstrandberg) [inline app/components/EventEditPicture/index.js]: Why not just use a Button here?

File: app/containers/Members/actions.js [hunk 3]
```diff
@@ -1,41 +1,39 @@
 import {
-  GET_MEMBERS_REQUEST,
-  GET_MEMBERS_SUCCESS,
+  GET_MEMBERS_REQUEST, GET_MEMBERS_SUCCESS,
```
Comment (aronstrandberg) [inline app/containers/Members/actions.js]: One import per line?

File: app/components/EventEdit/index.js (hunk 1 above)
Comment (jeppes) [reply app/components/EventEdit/index.js]: Will remove the limit for now

**Pull request merged**